### ISSUE TITLE
Add code comments to project templates

### ISSUE THREAD

**Issue by magi42** [vaadin/vaadin-gradle-plugin#4]
It is important that all project stubs are as clear as possible. While code should be self-documenting, it is also important to describe what is being done and why. Currently, the project stubs do not have any descriptive comments.

Add a comment line or two at least before each code section, or line, as is appropriate to clarify the intent of the line.

There should generally be an empty line before a comment line, except at the beginning of a code block.

For example:
```
/**
 * The main view of the application
 */
@Route("")
@Theme(Lumo.class)
@BodySize(height = "100vh", width = "100vw")
@Viewport("width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes")
@CssImport("./theme.css")
public class MyprojectView extends VerticalLayout {

    public MyprojectView() {
        // Set the CSS style name for the top-level element of the view
        setClassName("app-view");

        // Add a text label component on the top of the view
        Label hello = new Label("Hello Gradle app!");
        add(hello);

        // Add a button that does things when clicked
        Button button = new Button("This is a Vaadin Button, Click me!", event -> {
            // Change styling for the label when the button is clicked
            hello.setClassName("clicked");

            // Change the actual text
            hello.setText("Clicked! Next is a custom Polymer template button, Click it!");

            // Hide the original button
            event.getSource().setVisible(false);

            // Replace it with a custom button component
            add(new HelloButton());
        });
        add(button);
    }
}
```

**Comment by mvysny** [comment]
As discussed with @mstahv the correct place for these stubs is in the Vaadin IDE plugin. We will remove support for generating source code. Solved when #18 is implemented. Also see #5 .

**Comment by mvysny** [comment]
The project stubs will not be generated by the new plugin; the old plugin will be removed at some point. There is no point in fixing those project stubs. Marking as won't fix.
